src/main/java/com/musala/atmosphere/client/Device.java

```diff
@@ -1433,14 +1442,197 @@ public Boolean restoreWifiConnectionProperties() {
         return (Boolean) communicator.sendAction(RoutingAction.UNSHAPE_DEVICE);
     }
 
+    public void startLogcat() {
+        startLogcat(LOCAL_DIR, LogCatLevel.VERBOSE);
+    }
+
+    /**
+     * Starts local LogCat logging by log file path and {@link LogCatLevel} levels.
+     *
+     * @param logFilePath
+     *        - path to the log file where device log will be stored
+     * @param logLevels
+     *        - log levels to be set for filtering
+     */
+    public void startLogcat(String logFilePath, LogCatLevel... logLevels) {
+        String logcatParams = calculatesLogLevels(logLevels);
+        startLogcat(logFilePath, logcatParams);
+    }
+
+    /**
+     * Starts local LogCat logging by log file path, {@link LogCatLevel} levels and tag.
+     *
+     * @param logFilePath
+     *        - path to the log file where device log will be stored
+     * @param logLevel
+     *        - log level to be set for filtering
+     * @param tag
+     *        - tag for filtering used in combination with the given logLevel
+     */
+    public void startLogcat(String logFilePath, LogCatLevel logLevel, String tag) {
+        String logcatParams = logLevel.getLevelTagFilter(tag) + LogCatLevel.SILENT.getFilterValue();
+        startLogcat(logFilePath, " -s " + logcatParams);
+    }
+
     /**
-     * Clears the logacat from the device.
+     * Starts local LogCat logging by log file {@link LogCatLevel} level and tag. Uses the local file path as a default.
+     *
+     * @param logLevel
+     *        - log level to be set for filtering
+     * @param tag
+     *        - tag for filtering used in combination with the given logLevel
      */
-    public synchronized void clearLogcat() {
+    public void startLogcat(LogCatLevel logLevel, String tag) {
+        startLogcat(LOCAL_DIR, logLevel, tag);
+    }
+
+    /**
+     * Starts local LogCat logging by log file {@link LogCatLevel} levels. Uses the local file path as a default.
```

Comment:
In the method description be more client oriented. It is better to say something like : " The logcat file will be saved in ..." than "Uses the local file path as a default."